src/utils/convertFunc.js

```diff
@@ -0,0 +1,9 @@
+export function convertKevinToCelcius(tempDegree) {
+  return Math.round(tempDegree - 273.15)
```

Comment:
> should add fixed because there is no guarantee that `tempDegree` is rounded to 2 decimal places

use `Math.round()` to get the nearest integer (not rounding to 2 decimal places)